crates/adapters/src/static_compile/seroutput.rs

```diff
@@ -386,26 +490,30 @@ where
 
     fn step_key(&mut self) {
         self.cursor.step_key();
+        self.skip_zero_keys();
```

Comment:
what changed to make these necessary?

Reply:
We now output traces without fully consolidating them. Traces can contain values with 0 weights (due to weights in different batches canceling out)